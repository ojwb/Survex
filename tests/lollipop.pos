( Easting, Northing, Altitude )
(    0.00,     0.00,     0.00 ) 1
(    4.00,     0.00,     0.00 ) 2
(    2.00,    -0.00,     0.00 ) a
(    2.00,     3.00,     0.00 ) b
(    2.00,     5.00,     0.00 ) c
