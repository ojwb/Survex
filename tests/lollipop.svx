; pos=yes warn=0
*fix 1 0 0 0
*fix 2 4 0 0
1 A 3 090 0
A 2 3 090 0
A B 3 000 0
B C 2 000 0
C B 2 180 0
