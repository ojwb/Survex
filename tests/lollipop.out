
Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 5 survey stations, joined by 5 legs.
There is 1 loop.
Total length of survey legs =   13.00m (  11.00m adjusted)
Total plan length of survey legs =   13.00m
Total vertical length of survey legs =    0.00m
Vertical range = 0.00m (from c at 0.00m to c at 0.00m)
North-South range = 5.00m (from c at 5.00m to 1 at 0.00m)
East-West range = 4.00m (from 2 at 4.00m to 1 at 0.00m)
   2 1-nodes.
   1 2-node.
   2 3-nodes.
